README.md

```diff
@@ -14,3 +40,9 @@ Patent #17/337,413. Pending.
 
 This software is distributed under a particular license. Please see the
 [*LICENSE*](LICENSE) file for details.
+
+
+[FINTA]: ./doc/bibtex/Legarreta21_-_MIA_-_FINTA.bib "Filtering in tractography using autoencoders (FINTA)"
+[CINTA]: ./doc/bibtex/Legarreta22_-_MICCAI-CDMRI_-_CINTA.bib "Clustering in Tractography Using Autoencoders (CINTA)"
+[GESTA]: ./doc/bibtex/Legarreta22_-_arXiv_-_GESTA.bib "Generative sampling in tractography using autoencoders (GESTA)"
+[FIESTA]: ./doc/bibtex/Dumais22_-_arXiv_-_FIESTA.bib "FIESTA Autoencoders for accurate fiber segmentation in tractography"
```

Comment:
Typo: `FIESTA: `

Reply:
Well, the title I've put in arXiv takes a colon after the word FIESTA

Reply:
Ah, sorry, I should have paid attention. The bibtex was OK. Fixed in 98fa395.